js/layout-javascript/simple-list-code.js

```diff
@@ -73,17 +73,17 @@ function DynamicList(id, data, container) {
     }
 
     if (_.get(session, 'entries.dataSource.data')) {
-       _.extend(_this.myUserData, _.get(session, 'entries.dataSource.data'));
+      _.extend(_this.myUserData, _.get(session, 'entries.dataSource.data'));
     }
 
     // Start running the Public functions
     _this.initialize();
   });
-};
+}
```

Comment:
Find all other instances. I don't want to look through the PR to try and pick them all out.

```suggestion
};
```

Reply:
I went through all the files and double-checked the presence of ';'